libs/java/cert_refresher/src/main/java/com/oath/auth/KeyRefresher.java

```diff
@@ -160,6 +162,7 @@ private void scanForFileChanges() {
                         Thread.sleep(retryFrequency);
                     }
                 } catch (InterruptedException ignored) {
+                    LOGGER.error(ignored.getMessage());
```

Comment:
print error text with original exception

Reply:
Thanks, @pranavbhole. Incorporated the changes in https://github.com/AthenZ/athenz/pull/1526